src/regions/ca/info/ca-qc/reporting.md

```diff
@@ -1,5 +1,11 @@
 ## Québec: Self-Care Guide - COVID-19
 
+Quebec has developed a preliminary coronavirus [(COVID-19) evaluation tool](https://covid19.quebec.ca/evaluation/)
+.
```

Comment:
This `.` should be on the previous line :)

Reply:
Should be fixed now :). Thank you for your vigilance!